index.html

```diff
@@ -389,14 +389,15 @@ <h3>
               object with its {{WakeLockSentinel/type}} attribute set to
               |type|.
               </li>
-              <li>Let |success:boolean| be the result of awaiting <a>acquire a
-              wake lock</a> with |lock| and {{WakeLockType/"screen"}}:
-                <ol>
-                  <li>If |success| is `false` then reject |promise| with a
-                  {{"NotAllowedError"}} {{DOMException}}, and abort these
-                  steps.
-                  </li>
-                </ol>
+              <li>Invoke <a>acquire a wake lock</a> with |lock| and
+              {{WakeLockType/"screen"}}.
+                <aside class="note">
+                  The <a>acquire a wake lock</a> algorithm may ultimately be
+                  unable to acquire a lock from the operating system, but this
+                  is indistinguishable from a successful lock acquisition to
+                  avoid user fingerprinting (a failure to acquire a lock can
+                  indicate low battery levels, for example).
```

Comment:
We might want to call out explicitly that, aside from failure to satisfy the normative requirements when making a call to `request()` a wake lock acquisition should be considered advisory-only by the browser.